frontend/src/components/data-table/pagination.tsx

```diff
@@ -101,9 +102,27 @@ export const DataTablePagination = <TData,>({
   );
   const totalPages = table.getPageCount();
 
+  const pageSize = table.getState().pagination.pageSize;
+
   return (
     <div className="flex flex-1 items-center justify-between px-2">
-      <div className="text-sm text-muted-foreground">{renderTotal()}</div>
+      <div className="flex items-center gap-2">
+        <div className="text-sm text-muted-foreground">{renderTotal()}</div>
+        <div className="flex items-center gap-1 text-xs">
+          <NumberField
```

Comment:
i wonder if we should create opinionated "steps" e.g. 5/10/25/100, instead of a free-from input.

we could also do `[props.pageSize, 5, 10, 25, 100].sort()`

Reply:
we offer the flexibility with `mo.ui.table(page_size)`. as a end-user, you may just want to quickly increase/decrease. 

im not super opinionated that it needs to be a dropdown vs a number, but maybe we can poll some other folks?